samples/conda/README.md

```diff
@@ -0,0 +1,77 @@
+<!--
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# How to Create a Custom Conda Environment
+
+If you would like to run conda with the latest version of vLLM, you will need to create a
+a [custom execution environment](https://github.com/triton-inference-server/python_backend#creating-custom-execution-environments).
+This is because vLLM currently does not support the latest versions of CUDA in the Triton environment.
+Instructions for creating a custom execution environment with the latest vLLM version are below.
+
+## Step 1: Build a Custom Execution Environment With vLLM and Other Dependencies
+
+The provided script should build the package environment
+for you which will be used to load the model in Triton.
+
+Run the following command from this directory. You can use any version of Triton.
+```
+docker run --gpus all -it --rm -p 8000:8000 -p 8001:8001 -p 8002:8002 --shm-size=8G --ulimit memlock=-1 --ulimit stack=67108864 -v ${PWD}:/work -w /work nvcr.io/nvidia/tritonserver:23.09-py3 bash
```

Comment:
IIRC `nvcr.io/nvidia/tritonserver:23.09-py3` should have python backend in there, right?

Reply:
I think if the latest container (which is mentioned here) have python backend already, we can skip it's build in `.sh` and just create the environment. Latest python backend would be able to find `stub` in the default python backend location.

Reply:
Olga and I discussed this offline. Since the CUDA version differs, it is most likely still necessary to create the stubs.

Note: When the vLLM backend container ships, it would be good to update the above command to use that so users do not need to pull the Triton server container unnecessarily.

Reply:
I looked into docs after meetings, it seems like `CUDA` version should not be a trigger for a new `stub` : https://github.com/triton-inference-server/python_backend#building-custom-python-backend-stub

CC @Tabrizian and @tanmayv25 I would like to clarify the above. If we already ship container with `python` backend, we should be able to re-use without extra cmake build. I'll test it locally meanwhile

Reply:
Thanks for checking! Yeah, if it works on your end, we can simplify this. Feel free to make a commit to co-author this PR, if you would like, or let me know when it works and I'll remove those steps.

Reply:
Working on it

Reply:
These steps can be removed for 23.10 release in my opinion. We can just talk about building with `build.py`. If they need to build with different version of vLLM they can update the vLLM version from the version map. 
Or simply remove and re-installed vllm package in our shipped container. Let's just offer these two alternatives for simplicity.

Reply:
I think I agree with @tanmayv25, since we are moving into the direction of shipping container with conda environment, `conda` folder will become redundant + It would be easier to add it later as (and if) needed.

Reply:
They cannot do either of those. If vLLM releases another pinned vLLM version, they can update the version map or install it. However, if they want to use the latest vLLM version, it will be incompatible with the CUDA in the container (hence the need for Conda). Is that still acceptable?

I'm happy to remove this. I just want to make sure customers can still use whichever version of vLLM they want.